### ISSUE TITLE
Add a Contest Drop Down in Contest Info Tab 

### ISSUE THREAD

**Issue by rrojas350** [BrightSpots/rcv#596]
New drop down in the Contest Info tab that will set the rules for the contest, similar to Winning Rules, but across all tabs. This will load set perimeters in RCTab based off the type of RCV contest selected.

**Comment by chughes297** [comment]
Could we do this by including a set of pre-made configuration files in the install/downloadable?

**Comment by rrojas350** [comment]
We could include pre-made configuration files in the readme folder with documentation. At the very least, they could serve as examples of common configs. That's one option.

**Comment by tarheel** [comment]
Yeah, I think providing some template configs as part of the distribution would be a good first step here.
